src/vmm/src/lib.rs

```diff
@@ -655,6 +656,15 @@ impl Vmm {
             .map_err(VmmError::DeviceManager)
     }
 
+    /// Updates the rate limiter parameters for block device with `drive_id` id.
+    pub fn update_vhost_user_block_config(&mut self, drive_id: &str) -> Result<(), VmmError> {
```

Comment:
discussed offline, we would like to add a store metric to monitor how long this operation takes.